tools/txg/TXG/Types.hs

```diff
@@ -88,9 +89,14 @@ data TXCmd
   | RunSimpleExpressions TimingDistribution
   | PollRequestKeys ByteString
   | ListenerRequestKey ByteString
+  | SingleTransaction SingleTX
   deriving (Show, Eq, Generic)
   deriving anyclass (FromJSON, ToJSON)
 
+data SingleTX = SingleTX { _stxCmd :: Text, _stxChainId :: ChainId }
```

Comment:
Is there a check to make sure the chain id is a member of the peterson graph?

Reply:
Yes, let me do that.

Reply:
As it turns out: at parse-time, no. It would need to be an extra manual check somewhere at runtime.